packages.json

```diff
@@ -1,18 +1,18 @@
 [
     {
         "name": "mediation-add-header",
-        "version": "0.2.0"
+        "version": "0.3.0"
     },
     {
         "name": "mediation-json-to-xml",
-        "version": "0.2.0"
+        "version": "0.3.0"
     },
     {
         "name": "mediation-rewrite-resource-path",
-        "version": "0.2.0"
+        "version": "0.3.0"
     },
     {
         "name": "mediation-xml-to-json",
-        "version": "0.2.0"
+        "version": "0.3.0"
     }
 ]
```

Comment:
```suggestion
]

```

Reply:
No point in adding this though. This file is automatically updated through the workflow script. And it keeps dropping the final newline char when it does.